File: fibonacci/js/recursion.js
console.log(0);
console.log(1);

let count = 2;
let numberCount = 20;

function fibonacci(prev1, prev2) {
  if (count < numberCount) {
    const newFibo = prev1 + prev2;
    console.log(newFibo);
    prev2 = prev1;
    prev1 = newFibo;
    count++;
    fibonacci(prev1, prev2);
  } else {
    return;
  }
}

fibonacci(1, 0);

File: fibonacci/python/recursion.py
print(0)
print(1)
count = 2
numberCount = 20 # 20 numbers in the sequence

def fibonacci(prev1, prev2):
    global count
    if count < numberCount:
        newFibo = prev1 + prev2
        print(newFibo)
        prev2 = prev1
        prev1 = newFibo
        count += 1
        fibonacci(prev1, prev2)
    else:
        return

fibonacci(1,0)